index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>World Snooker Championship</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;0,900;1,400&display=swap"
+    rel="stylesheet" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous" />
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"
+    defer></script>
+  <script src="https://kit.fontawesome.com/9d6e6aaa13.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="page-header">
+    <section class="nav-bar">
+      <div class="page-logo">
+        <a href="#" class="navbar-brand">
+          <img width="60px" height="80px" src="img/main-icon.png" alt="icon">
+          World Snooker Championship 2022</a>
+      </div>
+      <nav class="nav-container">
+        <button id="menu-btn" type="button">
+          <i class="fa-solid fa-bars"></i>
+        </button>
+        <ul class="nav-menu">
+          <li class="active"><a href="#">Home</a></li>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li><a href="#program-section">Program</a></li>
+          <li><a href="#player-section">Players</a></li>
+          <li><a href="#partner-section">Partner</a></li>
+          <li class="nav-btn"><a href="#">Events</a></li>
+        </ul>
+      </nav>
+    </section>
+  </header>
+  <main>
+    <article class="headline-container" id="headline-section">
+      <section class="d-flex flex-column headline">
+        <h1>
+          <span class="greeting"> "Hello! Snooker World" </span>
```

Comment:
- [ ] The right semantic tag for this element is an `hx` tag as this is a subtitle. So using an h2, h3 e.t.c. is the best way. Kindly fix this. 
- [ ] This goes for the about page as well